src/plugins/displayLayout/components/BoxView.vue

```diff
@@ -71,10 +73,14 @@ export default {
     },
     computed: {
         style() {
-            return {
-                backgroundColor: this.item.fill,
-                border: '1px solid ' + this.item.stroke
-            };
+            if (this.itemStyle) {
```

Comment:
I am guessing that this logic is to support styling from the Display Layout Toolbar OR styling from the style tab of the conditional inspector? If so, we should only be defining styles in one place. Let's do it as a followup though. 

@charlesh88 I think we decided that we would move styling to the style inspector?

Reply:
Yes, that's right. created an issue for followup: #2756